test/unit/mocha.spec.js

```diff
@@ -64,6 +65,44 @@ describe('Mocha', function() {
         );
       });
     });
+
+    describe('when options.global is provided', function() {
+      it('should pass options.global to #globals()', function() {
+        // eslint-disable-next-line no-new
```

Comment:
I was told not to use those eslint instructions, but I see they are used all over this file. So I guess it's ok.

Reply:
Yeah I think since this is testing the constructor, it's unavoidable. Could assign the instance to `var mocha`, but then the test would have to reference that var in some way to avoid a different eslint flag : )